src/renderer/contexts/DependencyContext.tsx

```diff
@@ -692,6 +693,7 @@ export const DependencyProvider = memo(({ children }: React.PropsWithChildren<un
         supplier()
             .catch((error) => {
                 logOutput(String(error));
+                log.error(error);
```

Comment:
while i was at it, i noticed we weren't actually logging errors from this, only writing them to the terminal view component, so i fixed that as well